omod-common/src/main/java/org/openmrs/module/webservices/rest/web/RestUtil.java

```diff
@@ -65,17 +62,29 @@ public class RestUtil implements GlobalPropertyListener {
 	private static Log log = LogFactory.getLog(RestUtil.class);
 	
 	private static boolean contextEnabled = true;
+
```

Comment:
Since you decided to document the parameters, it would be great if you also included the method description. 😊